source/CommonJobs/CommonJobs.MVC.UI/Content/Evaluations/evaluations.css

```diff
@@ -0,0 +1,693 @@
+/**
```

Comment:
Hmm, good point. If you want, you can add a Nuget package to process SASS files and leave the original file, similarly to what we have done with LESS.

I would personally like to move all CSS to SCSS, but that's a personal preference, and probably not something that you guys will work on right now.